tests/pages/components/header.js

```diff
@@ -24,6 +24,10 @@ export default {
     scope: '[data-test-subscribe-button]',
   },
 
+  upgradeButton: {
```

Comment:
@libmartinito apologies if I wasn't clear in the Linear task - We don't need a separate subscribe vs. upgrade button. We just need the rename the button to "Upgrade" everywhere and additionally show it when a user isn't signed in